Remotion/Web/Core/Res/Themes/NovaViso/HTML/DropDownMenu.css

```diff
@@ -1,374 +1,207 @@
 /* Style applied to the DropDownMenu. */
 
-span.DropDownMenuContainer 
+span.DropDownMenuContainer
 {
-  display: block;
+  display: flex;
+  align-items: center;
   position: relative;
-  font-family: sans-serif;
+  height: var(--remotion-themed-min-height);
   width: 100%;
-  min-width: 18px;
-}
-
-tr.bocListDataRow span.DropDownMenuContainer,
-tr.bocListDataRowSelected span.DropDownMenuContainer
-{
-  display: inline-block;
 }
 
 span.DropDownMenuSelect
 {
-  display: block;
+  display: inline-flex;
   position: relative;
-  border: solid 1px #AAAAAA;
-  text-align: left;
-  vertical-align: top;
+  line-height: var(--remotion-themed-line-height);
+  width: 100%;
+  border: var(--remotion-themed-border);
+  border-radius: var(--remotion-themed-border-radius);
   white-space: nowrap;
-  cursor: hand;
   cursor: pointer;
-  line-height: 1.25em;
-  height: 1.4em;
 }
 
-td.buttonCell span.DropDownMenuSelect
+div.DropDownMenuOptions
 {
-  /* only when display just arrow */
-  display: inline-block;
+  /* int max for highest z-index possible */
+  z-index: 2147483647;
+  position: absolute;
 }
 
-span.DropDownMenuSelect.disabled
+ul.DropDownMenuOptions
 {
-  cursor: default;
-  border: 1px solid #BCBCBC;
+  box-shadow: var(--remotion-themed-box-shadow);
+  padding: var(--remotion-themed-padding);
+  list-style: none;
+  background-color: var(--color-disabled-background);
 }
 
-span.DropDownMenuSelect span
+span.DropDownMenuItemIcon
 {
-  left: 0;
-  vertical-align: middle;
+  width: 1.25em;
 }
 
-span.DropDownMenuSelect a.DropDownMenuLabel
+/* style separator */
+span.DropDownMenuSeparator
 {
-  position: relative;
-  left: 0.1em;
-  top: 0.1em;
-  margin-right: 25px;
-  margin-right: calc(22px + 0.1em);
-  margin-left: 4px;
-  border-style: none;
-  background-color: Transparent;
-  color: Black;
-  font-weight: bold;
-  height: 1.2em !important;
+  display: flex;
+  line-height: var(--remotion-themed-separator-height);
+  margin: var(--remotion-themed-margin) 0;
+  background-color: var(--color-contrast-separator);
 }
 
-span.DropDownMenuSelect a.DropDownMenuLabel:focus
+div.DropDownMenuStatus
 {
-  outline: none;
-  text-decoration: underline;
+  overflow: hidden;
+  /* int max for highest z-index possible */
+  z-index: 2147483647;
+  /* position and dimension are set via javascript */
+  max-height: 4em;
+  max-width: 30em;
 }
 
-span.DropDownMenuContainer span.disabled a.DropDownMenuLabel
+/* TODO: rework indicator when new design available */
+div.DropDownMenuStatus > div
```

Comment:
We can do cleanup/styling based on the exisitng properties and concepts.

introduce `color-alert-background` and `color-alert-border` and `color-alert-text` and `remotion-themed-alert-border` etc.

The icon can be done same as the others with 1.25em, padding can be standard padding.